examples/control_example.py

```diff
@@ -0,0 +1,45 @@
+""" Example that takes control of QTM, starts a measurment and stops after 10 seconds and saves the measurement to a file """
+
+import asyncio
+import logging
+
+
+import qtm_rt
+
+LOG = logging.getLogger("example")
+
+
+async def setup():
+    """ Main function that connects to the QTM server, starts a measurement, and saves the data to a file """
+    connection = await qtm_rt.connect("127.0.0.1")
+
+    if connection is None:
+        return -1
+
+    async with qtm_rt.TakeControl(connection, "password"):
+        state = await connection.get_state()
+        if state != qtm_rt.QRTEvent.EventConnected:
+            await connection.new()
+            try:
+                await connection.await_event(qtm_rt.QRTEvent.EventConnected, timeout=10)
+            except asyncio.TimeoutError:
+                LOG.error("Failed to start new measurement")
+                return -1
+        # Start the measurement
+        await connection.start()
+        await connection.await_event(qtm_rt.QRTEvent.EventCaptureStarted, timeout=10)
+
+        # Wait 10 seconds
+        await asyncio.sleep(10)
```

Comment:
Question: Shouldn't `await connection.await_event` be blocking, since you used the `await` keyword? If so, the sleep is unnecessary.

Reply:
The sleep is to do a 10 seconds recording, made the comment more clear.

Reply:
Nice!